src/components/nodes/startEvent/startEvent.vue

```diff
@@ -14,7 +14,20 @@
     @remove-node="$emit('remove-node', $event)"
     @add-node="$emit('add-node', $event)"
     @save-state="$emit('save-state', $event)"
-  />
+  >
+    <template v-slot:dropdown>
+      <div class="dropdown-container">
+        <button class="button" @click="dropdownOpen = !dropdownOpen">
+          <i class="fas fa-cog"/>
+        </button>
+
+        <select v-if="dropdownOpen" class="dropdown" @change="$emit('replace-node', node, $event.target.value)">
+          <option value=""/>
+          <option value="processmaker-modeler-start-timer-event">Start Timer Event</option>
+          <option value="processmaker-modeler-message-start-event">Message Start Event</option>
+        </select></div>
+    </template>
```

Comment:
I added this as a named slot just to get the initial implementation added quickly. It may be better to add this to the crown, especially since the other start events extend this component and so it will appear for all start events that way it is now.

The one advantage of this is that is eliminates the need for the crown config to pass down props and emit up events that it doesn't need to know about (in this case, it's just the `replace-node` event, though that could of course change).